cmd/detectExecuteScan.go

```diff
@@ -118,6 +118,13 @@ func runDetect(config detectExecuteScanOptions, utils detectUtils) error {
 	return utils.RunShell("/bin/bash", script)
 }
 
+func getDetectScript(config detectExecuteScanOptions, utils detectUtils) error {
+	if config.ScanOnChanges {
+		return utils.DownloadFile("https://raw.githubusercontent.com/blackducksoftware/detect_rescan/master/detect_rescan.sh", "detect.sh", nil, nil)
```

Comment:
I guess there is no *more official-looking* link available yet?

Reply:
Unfortunately not yet. Since it is a shell script, we have to download the source

Reply:
I already checked `https://detect.synopsys.com/detect_rescan.sh` but without success.